### PR TITLE
fix: support babel.config.js

### PR DESCRIPTION
**Pull request by fobbyal**
**What**: Added logic to recognize babel.config.js as a source of babel config

**Why**:  Latest babel documentation recommends babel.config.js (stating that babel itself uses this method). https://babeljs.io/docs/en/configuration

**How**: By adding additional !hasFile logic to the babel.js file

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by fobbyal** [comment]
I am not very clear on what you mean. 
1. Compiled then when using rollup only? 
2. how would i know which modules needs to be compiled? (via command-line parameters?)
3. Do we need to hold up this PR for the node_modules feature?

**Comment by kentcdodds** [comment]
I think it actually should just take an update to the rollup config to remove the `exclude` reference to node_modules.

**Comment by fobbyal** [comment]
i amended my commit. I do have one question though. Does this mean the build will get a performance hit since babel will try to compile everything? (Sorry i am no babel expert)

File: src/config/rollup.config.js [hunk 1]
```diff
@@ -115,7 +115,8 @@ const output = [
   },
 ]
 
-const useBuiltinConfig = !hasFile('.babelrc') && !hasFile('.babelrc.js') && !hasPkgProp('babel')
```
Comment (kentcdodds) [inline src/config/rollup.config.js]: Could you update this to allow for the `babel.config.js` as well?

Comment (fobbyal) [reply src/config/rollup.config.js]: oh sorry, I didn't catch that. I am not really a rollback user exactly. I am copying kcd-scripts to make something that feels more natural to me. Thanks for all the effort you put into the community!!!

**Pull request merged**